src/Microsoft.ServiceFabric.AspNetCore.Configuration/ServiceFabricConfigurationProvider.cs

```diff
@@ -43,6 +41,16 @@ public override void Load()
             this.LoadPackage(config);
         }
 
+        private void HandleNewPackage(ConfigurationPackage package)
+        {
+            // Load configuration from new package only if it is the ConfigPackage mapped to this provider
+            if (package.Description.Name == this.options.PackageName)
```

Comment:
Can I always expect to have Description available? Is it possible for this to be null (in a real, non-unit test setting)?

Reply:
Added a null check